api-reference/10 UI Components/dxPieChart/5 Series Types/CommonPieChartSeries/color.md

```diff
@@ -8,4 +8,124 @@ default: undefined
 Specifies a series color.
 
 ---
-Use this property to paint all pie slices in one color.
\ No newline at end of file
+You can use this property to paint all pie slices in one color.
+
+#include common-colorlist
+
+You can also specify a custom pattern or gradient instead of a plain color. Implement the [registerPattern()](/Documentation/ApiReference/Common/Utils/viz/#registerPatternoptions) or [registerGradient()](/Documentation/ApiReference/Common/Utils/viz/#registerGradienttype_options) method to get an id. Then, do the following:
```

Comment:
I would move the method call to the list too. 

```suggestion
You can also specify a custom pattern or gradient instead of a plain color. Follow the steps below: 

1. Call the [registerPattern()](/Documentation/ApiReference/Common/Utils/viz/#registerPatternoptions) or [registerGradient()](/Documentation/ApiReference/Common/Utils/viz/#registerGradienttype_options) method to obtain a fill ID.
```

Reply:
Noted